sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/workflowentity/WorkflowEntityImpl.scala

```diff
@@ -313,7 +315,8 @@ final class WorkflowEntityImpl(system: ActorSystem, val services: Map[String, Wo
                 executeStep.input,
                 executeStep.stepName,
                 service.messageCodec,
-                workflowContext)
+                timerScheduler,
+                SystemMaterializer(system).materializer.executionContext)
```

Comment:
Like this, for example? 

```suggestion
                system.dispatcher
```